### PR TITLE
New menu as navSide

### PR DESCRIPTION
**Pull request by sanlucca**
The new menu and its documentation
menu2

### CODE REVIEW

File: packages/scss/demo/templates/side-cards.html [hunk 1]
```diff
@@ -11,27 +11,27 @@
 	</head>
 	<body>
 		<main class="main">
-			<aside class="navSide">
-				<a href="#" class="navSide-link is-active">
+			<aside class="navSide mod-compact">
+				<a href="#" class="navSide-menuItem-item is-active">
```
Comment (jeremie-lucca) [inline packages/scss/demo/templates/side-cards.html]: This element is named like a child of a child. But it's only a direct child of navSide, so the class name must be wrong.

File: packages/scss/src/components/_nav-side.scss [hunk 2]
```diff
@@ -1,72 +1,226 @@
...
+
+	&::-webkit-scrollbar-thumb {
+		background-color: lighten(_component("navSide.primary-palette.color"), 20%);
+		border-radius: _component("navSide.scrollbar.border-radius");
+	}
 }
 
-.navSide-link {
-	color: white;
+// NECESSARY TO AVOID MOVEMENT WHEN THE SCROLLBAR APPEARS
```
Comment (jeremie-lucca) [inline packages/scss/src/components/_nav-side.scss]: Can you explain this ? :)

File: packages/scss/src/components/_nav-side.scss [hunk 3]
```diff
@@ -1,72 +1,226 @@
...
+	margin-left: 1em;
+	margin-right: 10px;
+	vertical-align: middle;
+}
+
+.navSide-item-alert {
+	background: _component("navSide.accent-palette.color");
 	border-radius: .5rem;
-	color: _color("text.default"); // lulu - i dont like it
+	color: _component("navSide.accent-palette.text"); // lulu - i dont like it
```
Comment (jeremie-lucca) [inline packages/scss/src/components/_nav-side.scss]: Lulu should like it now

File: packages/scss/src/theming/components/_nav-side.theme.scss [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+	"primary-palette": (
+		color: _color("primary"),
+		text: _theme("palettes.primary.text"),
+	),
+	"accent-palette": (
+		color: darken(_color("primary"), 5%),
+		text: _theme("palettes.primary.text"),
+	),
+	"offset-top": 0px,
+	"padding-top": 30px,
```
Comment (jeremie-lucca) [inline packages/scss/src/theming/components/_nav-side.theme.scss]: rem ?

File: packages/scss/src/theming/components/_nav-side.theme.scss [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+	"scrollbar": (
+		"width": 8px,
+		"border-radius": 10px,
+	),
+	"submenu": (
+		"max-height": 500px,
```
Comment (jeremie-lucca) [inline packages/scss/src/theming/components/_nav-side.theme.scss]: rem ?

File: packages/scss/src/theming/components/_nav-side.theme.scss [hunk 6]
```diff
@@ -1,5 +1,27 @@
...
+	),
+	"width": 240px,
```
Comment (jeremie-lucca) [inline packages/scss/src/theming/components/_nav-side.theme.scss]: rem ?

File: packages/scss/src/components/_nav-side.scss (hunk 2 above)
Comment (sanlucca) [inline packages/scss/src/components/_nav-side.scss]: It's a fixed width wrapper to keep its content from shifting when the scrollbar appear

**Comment by jeremie-lucca** [comment]
:smooch:

**Pull request merged**